### DISCUSSION TITLE
Deploy using Kubernetes Method

### DISCUSSION THREAD

**Discussion by andycampbell69** [ansibleguy76/ansibleforms#197] [Q&A] [answered]
Im trying to deploy AnsibleForms using the Kubernetes install method. The mysql node deploys ok, but the server node get the following error. "Warning  Evicted   kubelet  The node was low on resource: ephemeral-storage. Threshold quantity: 496815316, available: 443936Ki"
Ive tried to put limits on the server-deployment.yaml file but this has no effect.
Any ideas?

**Comment by ansibleguy76** [comment]
a pod is limited in storage. you are going over the limit.  that's what google tells me. 

apiVersion: v1
kind: Pod
metadata:
  name: frontend
spec:
  containers:
  - name: app
    image: images.my-company.example/app:v4
    resources:
      requests:
        ephemeral-storage: "2Gi"
      limits:
        ephemeral-storage: "4Gi"

**Comment by andycampbell69** [comment, marked as answer]
I ended up figuring it out, my /var mount point was filling up, you need at least 5GB available. Now I have another problem with the mysql tables being created....

**Reply by ansibleguy76** [reply]
the mysql tables issue is fixed in 5.0.4.  In 5.0.3 I rewrote the db-patching with async/await and dropped a few older db-patches, but it introduced a few bugs.
